dawn/README.md

```diff
@@ -3,82 +3,41 @@ Dawn <br/> <a target="_blank" href="http://semver.org">![Version][Version.Badge]
-<!--
-## Continuous Integration
+Where the `ADDITIONAL_FAGS` may include
+
+* `-DDAWN_BUNDLE_PYTHON=ON -DDAWN_PYTHON_EXAMPLES=ON` to also build Python examples that generate SIR <!--(IIR?!)--> in memory
+* `-DCLANG-FORMAT_EXECUTABLE=/usr/bin/clang-format` to add an additional target `make format` that formats the code into the formatting rules imposed by the project, as well as automatic code formatting for code generated by dawn.
+* `-DCMAKE_BUILD_TYPE=[Debug|Release|RelWithDebugInfo]` to set the build type
+* `-DDAWN_BUNDLE_JAVA=ON` to generate Java protobuf bindings
 
-### Linux
-|  Toolchain   | Config         |                                                     Status                                                   |
-|:-------------|:---------------|-------------------------------------------------------------------------------------------------------------:|
-| GCC 5.4      | Release        |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![GCC 5.4][GCC_54_Release.Badge]</a>          |
-| GCC 5.4      | RelWithDebInfo |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![GCC 5.4][GCC_54_RelWithDebInfo.Badge]</a>   |
-| GCC 6.3      | Release        |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![GCC 6.3][GCC_63_Release.Badge]</a>          |
-| GCC 6.3      | RelWithDebInfo |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![GCC 6.3][GCC_63_RelWithDebInfo.Badge]</a>   |
-| Clang 3.8    | Release        |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![Clang 3.8][Clang_38_Release.Badge]</a>        |
-| Clang 3.8    | RelWithDebInfo |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![Clang 3.8][Clang_38_RelWithDebInfo.Badge]</a> |
-| Clang 4.0    | Release        |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![Clang 4.0][Clang_40_Release.Badge]</a>        |
-| Clang 4.0    | RelWithDebInfo |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![Clang 4.0][Clang_40_RelWithDebInfo.Badge]</a> |
-| Clang 5.0    | Release        |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![Clang 5.0][Clang_50_Release.Badge]</a>        |
-| Clang 5.0    | RelWithDebInfo |  <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/dawn">![Clang 5.0][Clang_50_RelWithDebInfo.Badge]</a> |
--->
-## About
+## Testing
 
-This project is funded by the [PASCHA](http://www.pasc-ch.org/projects/2017-2020/pascha) project and developed by ETH Zurich and MeteoSwiss.
-Significant contributions were made by Fabian Thuering (Master Thesis), Carlos Osuna and Tobias Wicky. 
+There are two kinds of tests currently. Python tests which are launched by the `make install` step (if the necessary flags are set, see above) and tests using `Ctest`. To launch the latter:
```

Comment:
should we give the directory for this also? `/dawn/dawn/bundle/build && make install`